README.md

```diff
@@ -33,21 +33,22 @@ Or for the minified version
 Include jQuery and the plugin on a page.
 
 ```html
-$('elements').onScreen({
-   container: window,
-   direction: 'vertical',
-   doIn: function() {
-     // Do something to the matched elements as they come in
-   },
-   doOut: function() {
-     // Do something to the matched elements as they get off scren
-   },
-   tolerance: 0,
-   throttle: 50,
-   toggleClass: 'onScreen',
-   lazyAttr: null,
-   lazyPlaceholder: 'someImage.jpg',
-   debug: false
+var os = new OnScreen({
```

Comment:
This example should probably match what's on the http://silvestreh.github.io/onScreen/ website.

It's less useful when written in ES6. Not everyone uses ES6.